packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/quanticGeneratedAnswer.js

```diff
@@ -323,22 +387,26 @@ export default class QuanticGeneratedAnswer extends LightningElement {
     return this?.state?.citations;
   }
 
-  get hasCitations() {
-    return !!this.citations.length;
+  get shouldDisplayCitations() {
+    return !!this.citations.length && !this.isAnswerCollapsed;
```

Comment:
Never been a fan of `!!this.citations.length`, or maybe create a variable `const citationIsEmpty = !![...]` to make the return statement simpler to understand ? It's not too bad but complex binary operations in a return statement like are sometime hard to follow.